index.js

```diff
@@ -27,10 +27,15 @@ exports.handler = ({ body }, lambdaContext, callback) => {
 
   console.log(`Checking CLAs for PR ${context.webhook.pull_request.url}`);
 
-  githubRequest(getOrgConfig(context), clabotToken)
+  githubRequest(getOrgConfig(context), clabotToken, (response) => {
```

Comment:
There's a much neater way of doing this - you can catch a rejected promise, then return a value which is then resolved by handlers further down the chain. Here's a demo:

https://codepen.io/anon/pen/MmMyVy?editors=0011